scripts/checks/ruleof6/tests/test_false_positives.py

```
@@ -0,0 +1,243 @@
+
+        # Pattern from useEventSubscriptions.ts
+        dispatch_pattern = """type: 'message',
+          payload: {
+            content: `[DEBUG] EventBus: **${event.type}** | Source: ${event.source}`,
+            actor: 'system',
+          },
+          id: `debug-${event.type}-${Date.now()}`,
+          timestamp: event.timestamp ?? new Date(),
+          actor: 'system'"""
+
+        count = self.parser._count_arguments(dispatch_pattern)
```

Ruff (0.13.2)

Local variable `count` is assigned to but never used

Remove assignment to unused variable `count`

(F841)